test/recipes/30-test_evp_data/evpciph.txt

```diff
@@ -2078,6 +2078,36 @@ Operation = ENCRYPT
 Plaintext = 000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F20212223
 Ciphertext = A4DA23FCE6A5FFAA6D64AE9A0652A42CD161A34B65F9679F75C01F101F71276F15EF0D8D
 
+Title = SM4 test vectors
```

Comment:
What is the origin of these test vectors? Can we include a reference?

Reply:
Included a reference to `IETF draft-ribose-cfrg-sm4`.